study.md

```diff
@@ -134,15 +134,16 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+http is a protocol.  It is a procedure with formal instruction on transferring code between clients and servers.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+Clients are the local device that the user is using. A PC, iphone, ipad.
+Servers are hosted computers that communicate to the internet.
```

Comment:
This is correct although clients and servers are both communicating with the internet. Server is also communicating with the database.